app/script/view_model/content/InputBarViewModel.js

```diff
@@ -95,8 +96,19 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
       },
     });
 
+    this.mentionSuggestions = ko.pureComputed(() => {
+      const inputValue = this.input();
+      const incompleteMention = inputValue.match(/@(\w+)$/);
+      const candidates = this.conversationEntity().participating_user_ets();
+      if (incompleteMention) {
+        return this.searchRepository.searchUserInSet(incompleteMention[1], candidates);
```

Comment:
Let's name what you are pushing in here. => `const [, searchTerm] = incompleteMention`

Reply:
Again, this is just placeholder code, I don't think we should spend to much time on this as this is going to be replaced in the near future